app/containers/message/index.js

```diff
@@ -213,6 +217,8 @@ export default class MessageContainer extends React.Component {
 				broadcast={broadcast}
 				baseUrl={baseUrl}
 				useRealName={useRealName}
+				Message_Read_Receipt_Enabled={Message_Read_Receipt_Enabled}
```

Comment:
I know we don't follow the correct js naming pattern in all places (because of settings naming), but Message component is well done already.
Rename it to `isReadReceiptEnabled`.